src/adapters/wideorbit.js

```diff
@@ -0,0 +1,213 @@
+var bidfactory = require('../bidfactory.js'),
+    bidmanager = require('../bidmanager.js'),
+    utils = require('../utils.js'),
+    adloader = require('../adloader');
+
+var WideOrbitAdapter = function WideOrbitAdapter() {
+  var pageImpression = 'JSAdservingMP.ashx?pc={pc}&pbId={pbId}&clk=&exm=&jsv=1.0&tsv=1.0&cts={cts}&arp=0&fl=0&vitp=&vit=&jscb=window.parent.pbjs.handleWideOrbitCallback&url=&fp=&oid=&exr=&mraid=&apid=&apbndl=&mpp=0&uid=&cb={cb}&hb=1',
+      pageRepeatCommonParam = '&gid{o}={gid}&pp{o}=&clk{o}=&rpos{o}={rpos}&ecpm{o}={ecpm}&ntv{o}=&ntl{o}=&adsid{o}=',
+      pageRepeatParamId = '&pId{o}={pId}&rank{o}={rank}',
+      pageRepeatParamNamed = '&wsName{o}={wsName}&wName{o}={wName}&rank{o}={rank}&bfDim{o}={width}x{height}&subp{o}={subp}',
+      base = (window.location.protocol) + '//p{pbId}.atemda.com/',

```

Comment:
https is an expensive operation, also cost-wise for us. Is there some compatibility issue we overlook or similar? Please explain why we need to change this..